spec/integ/matrix-client-methods.spec.ts

```diff
@@ -1337,27 +1337,38 @@ describe("MatrixClient", function() {
         });
     });
 
-    describe("registerWithIdentityServer", () => {
-        it("should pass data to POST request", async () => {
-            const token = {
-                access_token: "access_token",
-                token_type: "Bearer",
-                matrix_server_name: "server_name",
-                expires_in: 12345,
-            };
+    describe("setPowerLevel", () => {
```

Comment:
It looks like you replaced an existing unrelated test?

Reply:
Yes, there were two completely identical tests, see: https://github.com/matrix-org/matrix-js-sdk/blob/develop/spec/integ/matrix-client-methods.spec.ts#L1316